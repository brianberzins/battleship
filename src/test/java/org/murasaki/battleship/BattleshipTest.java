package org.murasaki.battleship;

import org.approvaltests.Approvals;
import org.junit.jupiter.api.Test;

class BattleshipTest {

    @Test
    void testNewGame() {
        Board emptyBoard = new Board(5, 5);
        Battleship battleship = new Battleship(emptyBoard, emptyBoard);
        Approvals.verify(battleship);
    }

}
