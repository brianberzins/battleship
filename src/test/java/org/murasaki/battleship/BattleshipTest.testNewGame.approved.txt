next=player1
player1board=
     
     
     
     
     
player2board=
     
     
     
     
     
